addons/website/tests/test_views.py

```diff
@@ -1025,6 +1025,18 @@ def test_specific_view_translation(self):
         self.assertEqual(specific_view.with_context(lang='es_ES').arch, '<div>hola</div>',
                          "loading module translation should not remove specific translations that are not available on base view")
 
+        # Make sure updating a short list of languages does not destroy existing translations.
+        self.env['res.lang']._activate_lang('nl_NL')
+
+        self.env['ir.module.module']._load_module_terms(['website'], ['nl_NL'], overwrite=True)
+
+        specific_view.invalidate_model(['arch_db', 'arch'])
+        self.assertEqual(specific_view.with_context(lang='fr_BE').arch, '<div>salut</div>',
+                         "loading module translation copy translation from base to specific view")
+
+        self.assertEqual(specific_view.with_context(lang='es_ES').arch, '<div>hola</div>',
+                         "loading module translation should not remove specific translations that are not available on base view")
```

Comment:
I think you should adapt the assertions texts so it makes more sense no? Here is just a copy paste of the one above so on top of making it more difficult to debug if the test ever crashes I don't they are accurate assertions in this case: "loading module translation copy translation from base to specific view" -> was it really done a second time when loading NL terms?

Reply:
Here are the assertion texts and comments updates: https://github.com/odoo/odoo/pull/111302.

These changes have been included in the forward-port.